rental.rb

```diff
@@ -0,0 +1,16 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Please  the rental should add itself to the @book achieving the 'belongs-to' association` 
it should look something like this 
@book.rentals  << self 
- [ ] Please  the rental should add itself to the @person achieving the 'belongs-to' association` 
it should look something like this 
@person.rentals  << self